examples/image_classification.ipynb

```diff
@@ -910,10 +910,10 @@
     }
```

Comment:
<p>Update to paragraph: </p><p> </p><p>We'll load the image processor and model from the hub (here, we use the&nbsp;<a href="https://huggingface.co/docs/transformers/model_doc/auto#transformers.AutoModelForImageClassification" target="_blank">Auto Classes</a>, which will make sure the appropriate classes will be loaded automatically based on the&nbsp;<code>config.json</code>&nbsp;and&nbsp;<code>preprocessor_config.json</code>&nbsp;files of the repo on the hub):</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>